### DISCUSSION TITLE
Which American president had parakeets named “Bluebell” and Marybelle”?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#152] [Galaxy Trivia]
**Hint:** NY airport named after him

**Comment by torhaa1** [comment]


**Reply by ParanoidUser** [reply]
You did it! 😋 Congrats on your well deserved [Galaxy Brain](https://github.com/t-haakens?tab=achievements&achievement=galaxy-brain) badge!
If you're willing to be a part of the history, feel free to open PR and add your name to the [change log](https://github.com/ParanoidUser/yolo/blob/main/docs/CHANGELOG.md).
